### PR TITLE
[BEAM-228] Mergebot initial commit

### PR DESCRIPTION
**Pull request by jasonkuster**
Be sure to do all of the following to help us incorporate your contribution
quickly and easily:

 - [x] Make sure the PR title is formatted like:
   `[BEAM-<Jira issue #>] Description of pull request`
 - [x] Make sure tests pass via `mvn clean verify`. (Even better, enable
       Travis-CI on your fork and ensure the whole test matrix passes).
 - [x] Replace `<Jira issue #>` in the title with the actual Jira issue
       number, if there is one.
 - [x] If this contribution is large, please file an Apache
       [Individual Contributor License Agreement](https://www.apache.org/licenses/icla.txt).

---

This is the initial commit for a mergebot for Beam.

### CODE REVIEW

**Comment by jasonkuster** [comment]
R: @davorbonaci

File: .travis.yml [hunk 1]
```diff
@@ -33,4 +33,4 @@ install:
 
 script:
   - travis_retry mvn -B $MAVEN_OVERRIDE install -U
-  - travis_retry travis/test_wordcount.sh
```
Comment (davorbonaci) [inline .travis.yml]: let's do `testing` instead of `tools`.

Comment (jasonkuster) [inline .travis.yml]: Done

Comment (davorbonaci) [inline .travis.yml]: Not pushed, seems like.

Comment (jasonkuster) [inline .travis.yml]: Now is!

Comment (davorbonaci) [inline .travis.yml]: Can you get rid of README.md when you are there?

**Pull request closed without merging**